src/handler.rs

```diff
@@ -1,7 +1,82 @@
+use lunatic::function::reference::Fn as FnPtr;
+use lunatic::function::FuncRef;
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+
 use crate::extract::{FromOwnedRequest, FromRequest};
 use crate::response::IntoResponse;
 use crate::{RequestContext, Response};
 
+/// Implemented for process-safe [`Handlers`](Handler).
+///
+/// Submillisecond handles every request in a separate [lunatic
+/// process](Process), and lunatic's processes are sandboxed. This means that no
```

Comment:
This link doesn't seem to work. Might have to use the full URL since it's an external crate?

Reply:
I think it only doesn't work in rust-analyser, but does in the docs generated?

Reply:
Actually, I removed the import and forgot about it. Should be ok now.